Arch Linux: Permission denied: '/usr/lib/python3.6/site-packages/sip.so'

TL;DR `Makefile` row `python3 -m pip install -r requirements.txt` should also have the `--user` flag since it's being run as normal user. PR submitted.

## Expected Behavior

```
git clone https://github.com/Codaone/DEXBot.git dexbot
cd dexbot   
make install-user
```

Successful install

## Actual Behavior

`PermissionError: [Errno 13] Permission denied: '/usr/lib/python3.6/site-packages/sip.so'`

Full log:

```bash
krister ➜  dexbot git:(master) make install-user       
python3 -m pip install -r requirements.txt

[...redacted]

https://files.pythonhosted.org/packages/fe/fd/f63226be4aeebcac65d5f7e882bd00c8465ce883c1a4d16b18bd4ae9086e/altgraph-0.15-py2.py3-none-any.whl
Installing collected packages: sip, pyqt5, pyqt-distutils, click, click-datetime, future, pefile, altgraph, macholib, pyinstaller
Exception:
Traceback (most recent call last):
  File "/usr/lib/python3.6/site-packages/pip/basecommand.py", line 215, in main
    status = self.run(options, args)
  File "/usr/lib/python3.6/site-packages/pip/commands/install.py", line 342, in run
    prefix=options.prefix_path,
  File "/usr/lib/python3.6/site-packages/pip/req/req_set.py", line 784, in install
    **kwargs
  File "/usr/lib/python3.6/site-packages/pip/req/req_install.py", line 851, in install
    self.move_wheel_files(self.source_dir, root=root, prefix=prefix)
  File "/usr/lib/python3.6/site-packages/pip/req/req_install.py", line 1064, in move_wheel_files
    isolated=self.isolated,
  File "/usr/lib/python3.6/site-packages/pip/wheel.py", line 345, in move_wheel_files
    clobber(source, lib_dir, True)
  File "/usr/lib/python3.6/site-packages/pip/wheel.py", line 323, in clobber
    shutil.copyfile(srcfile, destfile)
  File "/usr/lib/python3.6/shutil.py", line 121, in copyfile
    with open(dst, 'wb') as fdst:
PermissionError: [Errno 13] Permission denied: '/usr/lib/python3.6/site-packages/sip.so'
You are using pip version 9.0.1, however version 10.0.1 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
make: *** [Makefile:20: pip] Error 2
```

## Specifications

- Version: `docs/conf.py:version = '0.0.1'`
- python-pip 9.0.1-3
- OS: Arch Linux

## Solution and the steps i took

1. Upgraded pip with `pip install --upgrade --user pip`. (not sure if needed, but it was my first clue) from version 9 to 10.
2. Now `make` gave same error, but also suggesting to use the `--user` flag.
3. **Edited Makefile and added `--user` to `python3 -m pip install --user -r requirements.txt`.**
4. Install is successful.

--- a/Makefile
+++ b/Makefile
@@ -19,16 +19,22 @@ clean-pyc:
 pip:
 	python3 -m pip install -r requirements.txt
 
+pip-user:
+	python3 -m pip install --user -r requirements.txt
+
 lint:
 	flake8 dexbot/
 
 build: pip
 	python3 setup.py build
 
+build-user: pip-user
+	python3 setup.py build
+
 install: build
 	python3 setup.py install
 
-install-user: build
+install-user: build-user
 	python3 setup.py install --user
 
 git:

